packages/cli/test/unit/ActiveExecutions.test.ts

```diff
@@ -0,0 +1,92 @@
+import { ActiveExecutions, IWorkflowExecutionDataProcess, Db } from '../../src';
+import { mocked } from 'jest-mock';
+import PCancelable from 'p-cancelable';
+import { IRun } from 'n8n-workflow';
+
+const FAKE_EXECUTION_ID = '15';
+const FAKE_SECOND_EXECUTION_ID = '20';
```

Comment:
This would work fine for some of the test cases but `Should add execution to active execution list` relies on checking the execution ID (if it matches) and the value is returned from the mocked promise.

Therefore, for the sake of consistency I think I'll be leaving it like this.

Reply:
Definitely, matter of preference.